Engine/source/console/debugOutputConsumer.h

```diff
@@ -27,8 +27,8 @@
 
 //#define TORQUE_LOCBUILD
 
-#if !defined(TORQUE_DEBUG) && defined(TORQUE_OS_XENON) && !defined(TORQUE_LOCBUILD)
-#define DISABLE_DEBUG_SPEW
+#if !defined(TORQUE_DEBUG) && !defined(TORQUE_LOCBUILD)
+   #define DISABLE_DEBUG_SPEW
```

Comment:
I guess it'd largely depend on the circumstance LOCBUILD is flagged. It's a separate flag, so I wasn't sure about trimming it as well if there's some logic behind it. If not though, then yeah could probably just trim that entire bit after all.